### ISSUE TITLE
SuperScript and SubScript formatting broken.

### ISSUE THREAD

**Issue by aswin-sw** [froala/react-froala-wysiwyg#207]
if end of a line(sentence) is formatted as superscript or subscript, then it stays that way for the rest of the line, unless we break the line.

P.S: ![bug](https://www.loom.com/share/8316944f7c3d41979a8b537aae5068e4)

**Comment by dejanmartinovic** [comment]
The fix has already been added to the master branch, you can still access it from https://cdn.jsdelivr.net/gh/froala/wysiwyg-editor@master/js/froala_editor.pkgd.min.js. 
You could test it here: https://jsfiddle.net/dejan_m/Lha56d3p/

**Comment by aswin-sw** [comment]
@dejanmartinovic the version in npm package registry seems to be different from the one here. Can you please tell me if this package is being published to npm?

**Comment by dejanmartinovic** [comment]
@aswin-sw Of course. I'm just saying that the issue is already fixed and you could reference it from https://cdn.jsdelivr.net/gh/froala/wysiwyg-editor@master/js/froala_editor.pkgd.min.js. The current version is 3.0.6 and this fix will be available in the next one (3.1.0). It should be released in the next couple of days

**Comment by aswin-sw** [comment]
@dejanmartinovic its more than 3 weeks now. will there a release anytime soon?
